robot/Cumulus/resources/locators_45.py

```diff
@@ -110,7 +110,7 @@
     },
     'payments':{
         'date_loc':"//*[@id='pmtTable']/tbody/tr/td[3]/div//input",       
-        'no_payments':'//tbody/tr/td[3]',
+        'no_payments':'//tbody/tr[./th//a[contains(@title,"PMT")]]/td[3]',
```

Comment:
Is this the locator for Payments() link in the Payments() component?

Reply:
this locator is for payment amounts on payments page that appears after clicking on View all button payments related list